cmd/configure.ac

```diff
@@ -77,6 +77,17 @@ AC_ARG_ENABLE([seccomp],
     esac], [enable_seccomp=yes])
 AM_CONDITIONAL([SECCOMP], [test "x$enable_seccomp" = "xyes"])
 
+# Allow to build with SELinux support by calling:
```

Comment:
Reexec idea: I'd like to depend on selinux and just optionally enable it (or not) somehow. This doesn't have to happen now but it would would strongly simplify the set of possible builds of snap-confine that we must consider. Ideally it would be just (exactly) one so that we can really support re-exec everywhere.

Reply:
```
    if (is_selinux_enabled() < 1) {
        debug("SELinux not enabled");
        return 0;
    }
```
This should do the trick. AA has similar functionality, so in theory we should be able to reexec on either AA or SELinux enabled systems (only if it wasn't for the mess caused by paths).

Reply:
That was not my point: can we just. not have the "--enable-selinux" flag at all and _always_ link to it?

Reply:
There are distros out there that don't support SELinux but have snapd available (Arch for instance). So this would have to degrade gracefully. Then we wouldn't know when you actually wanted SELinux enabled but it wasn't because libselinux is missing.

I think we could revisit this once we actually have an idea of making more universal use of SELinux support.